autobisect/build_manager.py

```diff
@@ -79,11 +79,12 @@ def remove_old_builds(self) -> None:
             for build_path in builds:
                 if self.current_build_size < self.config.persist_limit:
                     break
-
                 res = self.db.cur.execute(
-                    "SELECT * FROM in_use, download_queue "
-                    "WHERE in_use.build_path = ? OR download_queue.build_path = ?",
-                    (str(build_path), str(build_path)),
+                    "SELECT 1 FROM ("
```

Comment:
nit: `COUNT(1)` would be clearer and checking the result for 0/1 would be clearer to me, but up to you.